### DISCUSSION TITLE
why Angular support is lagging now?

### DISCUSSION THREAD

**Discussion by robertIsaac** [nrwl/nx#22303] [General]
previously supporting new Angular version as done in a matter of days
now Angular 17.3 is shipped while Angular 17.2 is still not supported in nx
I hope you will support both angular 17.3 and 17.2 in nx 18.1

**Comment by maartentibau** [comment]
Looking in the docs, they aren't really lagging behind, because the version is supported if you look at the version matrix.

https://nx.dev/nx-api/angular/documents/angular-nx-version-matrix

I assume that since Angular decided to release these 3 minor versions quite rapidly it's quite hard for the Nx team to keep up with that pace. But I don't think it would case any breaking changes if you updated the Angular specific dependencies manually.

**Reply by robertIsaac** [reply]
If there are no breaking changes, then why don't nx support angular with major versions instead of minor?
I mean to support any Angular version 17, 18 instead of 17.1, 17.2, ... Etc

Because now npm throw error by default
